pom.xml

```diff
@@ -391,6 +392,27 @@
         <scope>provided</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <groupId>io.opencensus</groupId>
+        <artifactId>opencensus-api</artifactId>
+        <version>${opencensus.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.opencensus</groupId>
+        <artifactId>opencensus-impl</artifactId>
+        <version>${opencensus.version}</version>
+      </dependency>
```

Comment:
Do we really need explicitly depend on this? Aren't they pulled in by other 2 deps?

Reply:
I don't think so.[Ref](https://opencensus.io/quickstart/java/metrics/#import-metrics-packages)

Reply:
`-impl` actually depends on `-api`, so we can remove at least it: https://search.maven.org/artifact/io.opencensus/opencensus-impl/0.31.0/jar